lkcrypto.c

```diff
@@ -778,6 +780,9 @@ static int __init modinit(void)
 
 static void __exit modexit(void)
 {
+	complete(&cipher_needs_init);
+	complete(&cipher_aead_needs_init);
+	complete(&hasher_needs_init);
```

Comment:
Similar comment for these — do we need to check that they haven't already been completed, or is this legit as-is?

Reply:
Internal functions in `complete` tell me close to nothing, but looking at kernel sources and docs I cannot find find any guards for checking if this has been completed